test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Migration/MigrationServiceTests.cs

```diff
@@ -42,6 +42,7 @@ public async Task TestHandleMigrationAssessmentRequest()
         }
 
         [Test]
+        [Ignore("Disable failing test")]
```

Comment:
this migration test isn't written correct, as the result return value isn't correctly being captures (and the service call is throwing an exception).  Disabling to let the owner team investigate.